bika/lims/browser/header_table.py

```diff
@@ -119,6 +119,11 @@ def render_field_view(self, field):
                            'html': ''}
             elif field.getType().lower().find('datetime') > -1:
                 value = field.get(self.context)
+                if value is None:
+                    attr = "get{}".format(fieldname)
+                    if hasattr(self.context, attr):
+                        attr = getattr(self.context, attr)
+                        value = attr()
```

Comment:
The call to the function should have priority over the value stored in the Schema's field. Also, if there is a Schema's field, we can assume a getter by default, so I think the following would be better:

```python
elif field.getType().lower().find('datetime') > -1:
    attr = "get{}".format(fieldname)
    value = getattr(self.context, attr)()
```

Nevertheless, please read the comment below re `DatePublishedViewer` and `DatePublished` fields, cause I think this attr workaround will not be needed.